### PR TITLE
Translate Polish comments to english

### LINKED ISSUE #80: 🌍 Translate the polish comments
There are few comments in zinit's codebase that are in polish.

We need to:
- Find them! (is there a regex for polish words? :D)
- Translate them!

Examples:
https://github.com/zdharma-continuum/zinit/blob/5e3e66818a0bec93a3ae6193273e7c2c78125a15/zinit.zsh#L255-L257

### PR DESCRIPTION
**Pull request by alichtman**
Fix #80

### CODE REVIEW

File: zinit.zsh [hunk 1]
```diff
@@ -1625,7 +1623,7 @@ builtin setopt noaliases
     (( ${+ICE[notify]} == 1 )) && { [[ $___retval -eq 0 || -n ${(M)ICE[notify]#\!} ]] && { local msg; eval "msg=\"${ICE[notify]#\!}\""; +zinit-deploy-message @msg "$msg" } || +zinit-deploy-message @msg "notify: Plugin not loaded / loaded with problem, the return code: $___retval"; }
     (( ${+ICE[reset-prompt]} == 1 )) && +zinit-deploy-message @___rst
 
-    # Wycofaj funkcję `m`.
+    # Withdraw the `m` function.
```
Comment (pschmitt) [inline zinit.zsh:1626]:
```suggestion
    # Unset the `m` function.
```

File: zinit.zsh [hunk 2]
```diff
@@ -1903,22 +1901,22 @@ builtin setopt noaliases
 .zinit-formatter-pid() {
     builtin emulate -L zsh -o extendedglob
 
-    # Zapamiętaj krańcowe białe znaki.
+    # Remember extreme whitespace
     local pbz=${(M)1##(#s)[[:space:]]##}
     local kbz=${(M)1%%[[:space:]]##(#e)}
-    # Usuń skrajne białe znaki.
+    # Remove extreme whitespace
```
Comment (pschmitt) [inline zinit.zsh:1907]:
```suggestion
    # trim whitespace
```

File: zinit.zsh [hunk 3]
```diff
@@ -1903,22 +1901,22 @@ builtin setopt noaliases
 .zinit-formatter-pid() {
     builtin emulate -L zsh -o extendedglob
 
-    # Zapamiętaj krańcowe białe znaki.
+    # Remember extreme whitespace
```
Comment (pschmitt) [inline zinit.zsh:1904]:
```suggestion
    # Save whitespace location
```

File: zinit.zsh [hunk 4]
```diff
@@ -1903,22 +1901,22 @@ builtin setopt noaliases
...
-    # Zamień przynajmniej jeden znak na niełamalną spację, ponieważ z
-    # powodu problemów z implementacją krańcowe białe znaki są gubione…
+    # Replace at least one character with an unbreakable space, because
+    # due to implementation problems, marginal whitespace is lost ...
     pbz=${pbz/[[:blank:]]/ }
     local kbz_rev="${(j::)${(@Oas::)kbz}}"
     kbz="${(j::)${(@Oas::)${kbz_rev/[[:blank:]]/ }}}"
 
-    # Dostaw spowrotem krańcowe białe znaki.
+    # Add extreme whitespace in
```
Comment (pschmitt) [inline zinit.zsh:1919]:
```suggestion
    # Re-add whitespace
```

File: zinit.zsh [hunk 5]
```diff
@@ -1978,28 +1976,27 @@ builtin setopt noaliases
     fi
     local append influx in_prepend
     if [[ $2 == (b|u|it|st|nb|nu|nit|nst) ]]; then
-        # Powtórzenie kodu aby zabezpieczyć ewentualne krańcowe białe znaki
-        # oraz aby umożliwić akumulację tego kodu z innymi.
+        # Code repetition to protect any extreme whitespace
```
Comment (pschmitt) [inline zinit.zsh:1979]:
```suggestion
        # Code repetition to preserve any leading/trailing whitespace
```

File: zinit.zsh [hunk 6]
```diff
@@ -1978,28 +1976,27 @@ builtin setopt noaliases
...
         if [[ $ZINIT[__last-formatter-code] != (…|ndsh|mdsh|mmdsh|-…|lr|rst|nl|) ]]; then
             in_prepend=$ZINIT[col-$ZINIT[__last-formatter-code]]
             influx=$ZINIT[col-$ZINIT[__last-formatter-code]]
         fi
-        # W przeciwnym razie brak akcji – tylko skopiowanie
-        # tego kodu bez koloru.
+        # Otherwise no action - only copy of this code without color.
     else
-        # Zakończenie aktywności kodu escape.
+        # Termination of escape code activity.
```
Comment (pschmitt) [inline zinit.zsh:1990]:
```suggestion
        # End of escaping logic
```

File: zinit.zsh [hunk 7]
```diff
@@ -1978,28 +1976,27 @@ builtin setopt noaliases
...
-    # Skonstruuj tekst.
+    # Construct the text.
     REPLY=$in_prepend${ZINIT[col-$2]:-$1}$influx$3$append
 
-    # Zamień nowe linie na znaki, które działają tak samo ale nie są
-    # usuwane w podstawieniu $( … ) – vertical tab 0xB ↔ 13 w systemie
-    # oktagonalnym połączone z powrotem karetki (015).
+    # Replace new lines with characters that work the same but are not
+    # deleted in the substitution $ (...) - vertical tab 0xB ↔ 13 in the system
+    # octagonal connected back carriage (015).
```
Comment (pschmitt) [inline zinit.zsh:1999]: Uh...?

File: zinit.zsh [hunk 8]
```diff
@@ -2027,12 +2024,12 @@ $(.zinit-main-message-formatter "$match[6]" "$match[7]" "$match[8]"; \
 $match[7]}:-${ZINIT[__last-formatter-code]}}}:+}}}//←→}
 
 
-    # Przywróć domyślny kolor na końcu wiadomości.
+    # Restore the default color at the end of the message.
```
Comment (pschmitt) [inline zinit.zsh:2027]:
```suggestion
    # Reset color attributes at the end of the message
```

**Review by pschmitt — CHANGES_REQUESTED** [review]
Nice, thanks!

I have a few suggestions though - see my comments.

File: zinit.zsh (hunk 7 above)
Comment (alichtman) [reply zinit.zsh:1999]: Yeah, no idea why we're doing this so I literally just copy+pasted the comment from Google Translate.

Comment (alichtman) [reply zinit.zsh:1999]: Functionally, it replaces `\013` with `\015`, but WHY? idk

Comment (pschmitt) [reply zinit.zsh:1999]: hm. Let's deal with that another time :)

**Pull request merged**